docs/language-server/protocol-language-server.md

```diff
@@ -351,6 +352,14 @@ interface Add {
 interface Remove {
   // suggestion entry id
   id: number;
+}
+
+interface Modify {
+  // suggestion entry id
+  id: number;
+  // new return type
+  returnType: String;
```

Comment:
Why can we only modify the return type?

Reply:
It updates the entry with the runtime type. The modification was initially implemented as Remove(id) and Add(suggestion) because it is what we get on the runtime side. Now I realized that we should be able to create the Modify updates in the language-server. I'll try to implement it today

Reply:
That makes sense to me, but surely it makes more sense to be able to modify _any_ part of the entry if we're allowing modifications.